docs/docsite/rst/user_guide/playbooks_intro.rst

```diff
@@ -406,6 +147,23 @@ Here's an example handlers section::
             name: apache
             state: restarted
 
+Controlling when handlers run
+-----------------------------
+
+By default, handlers run after all the tasks in a particular play have been completed. This approach is efficient, because the handler only runs once, regardless of how many tasks notify it. For example, if multiple tasks update a configuration file and notify a handler to restart apache, Ansible only bounces apache once to avoid unnecessary restarts.
```

Comment:
```suggestion
By default, handlers run after all the tasks in a particular play have been completed. This approach is efficient, because the handler only runs once, regardless of how many tasks notify it. For example, if multiple tasks update a configuration file and notify a handler to restart Apache, Ansible only bounces Apache once to avoid unnecessary restarts.
```